projects/ngx-interactive-paycard-demo/src/app/app.component.html

```diff
@@ -1,15 +1,16 @@
 <div class="wrapper">
-    <ngx-interactive-paycard
+  <ngx-interactive-paycard
     [chipImgPath]="'./assets/chip.png'"
-    [logoImgPath]="'./assets/logo.png'"
+    [logoImgPath]="brand"
     [frontBgImgPath]="'./assets/SplitShire1.jpg'"
     [backBgImgPath]="'./assets/SplitShire3.jpg'"
     [cardNumberFormat]="cardNumberFormat"
     [cardNumberMask]="cardNumberMask"
     [cardLabels]="cardLabel"
     [formLabels]="formLabel"
     (submitEvent)="onSubmitEvent($event)"
-    (submitChanges)="showChanges($event)"
```

Comment:
Note that this renaming is a breaking change.

Reply:
Any suggestions? I changed the name of the method because it didn't fit, it wasn't a Submit event, it was a return event.

Reply:
Okay, I agree that this naming is better. I'll update the next library version accordingly.